ims/exception/file_system_exceptions.py

```diff
@@ -41,6 +41,21 @@ def __str__(self):
         return self.name + " is Busy (Could have Clones)"
 
 
+class SnapshotBusyException(ImageBusyException):
+    """
+    This exception will be raised if user removes a protected snapshot
+    """
+    @property
+    def status_code(self):
+        ImageBusyException.status_code(self)
```

Comment:
@chemistry-sourabh , I tried your suggestion. It seems that we can't remove status_code part. I think the reason is that status_code is property. So I change code, and test it again. It works well